README.md

```diff
@@ -87,14 +112,13 @@ We demonstrate running an nginx container within the same pod as the aladdin-dem
       nginx:
         use: false
         port: 8001
-Change the `use` field to `true`. This is all you need to do to see nginx work, you can verify this by restarting the app with `aladdin restart`. Navigate to the aladdin-demo service pod in the Kubernetes dashboard and you should be able to see two containers running. Read on for how we did it. 
+        
+Set the `use` field to `true`. This is all you need to do to see nginx work, you can verify this by restarting the app with `aladdin restart`. Navigate to the aladdin-demo service pod in the Kubernetes dashboard and you should be able to see two containers running. Read on for how we did it. 
 
```

Comment:
I'm thinking you should just need to do `aladdin start` here rather than `aladdin restart`